### PR TITLE
Add support for token v2

### PR DESCRIPTION
**Pull request by kent-white**
The biggest change here was moving inline graphql queries to use the sdk queries that support token v2.

This is a start but there are some things missing from the SDK that need to be updated. Specifically
- Support for token v2 activities
- Support for sorting indexer queries? (Or at least better default sorting)
- Support for get owned tokens by token_data_id
- TYPES from Indexer PLEASE

### CODE REVIEW

File: src/pages/Token/Tabs/OverviewTab.tsx [hunk 1]
```diff
@@ -72,33 +51,33 @@ type OverviewTabProps = {
...
-          }
+          title={"Collection Name:"}
+          value={data?.current_collection?.collection_name}
         />
-        <ContentRow title={"Royalty:"} value={getRoyalty(data)} />
         <ContentRow
-          title={"Royalty Payee:"}
+          title={"Creator:"}
           value={
-            <HashButton hash={data?.payee_address} type={HashType.ACCOUNT} />
```
Comment (kent-white) [inline src/pages/Token/Tabs/OverviewTab.tsx:92]: royalty isn't a thing in the token v2 queries, will ask Bowen

File: src/ExplorerRoutes.tsx [hunk 2]
```diff
@@ -57,11 +57,8 @@ export default function ExplorerRoutes() {
           <Route path=":height/:tab" element={<BlockPage />} />
         </Route>
         <Route path="/token">
-          <Route path=":tokenId/:propertyVersion" element={<TokenPage />} />
-          <Route
-            path=":tokenId/:propertyVersion/:tab"
-            element={<TokenPage />}
-          />
```
[the remark this answers is not in the document]
Comment (kent-white) [reply src/ExplorerRoutes.tsx:64]: good point!

**Review by 0xmaayan — COMMENTED** [review]
ts sdk v1.16.0 is out and includes all missing queries/types for this PR

File: src/global-config/GlobalConfig.tsx [hunk 3]
```diff
@@ -21,6 +22,8 @@ export type GlobalState = {
   readonly network_value: string;
   /** derived from network_value */
   readonly aptos_client: AptosClient;
+  /** derived from network_value */
+  readonly indexer_client?: IndexerClient;
```
[the remark this answers is not in the document]
Comment (kent-white) [reply src/global-config/GlobalConfig.tsx:26]: oo very nice, maybe in a followup

**Comment by kent-white** [comment]
Probably can improve the indexer type @0xmaayan but no rush

File: src/pages/Token/Component/ActivitiesTable.tsx [hunk 4]
```diff
@@ -138,7 +142,7 @@ function ActivityHeaderCell({column}: ActivityHeaderCellProps) {
 }
 
 type ActivitiesTableProps = {
-  activities: any[]; // TODO: add graphql data typing
+  activities: Token_Activities_V2[]; // TODO: add graphql data typing
```
Comment (kaw2k) [inline src/pages/Token/Component/ActivitiesTable.tsx:145]: Im getting an error locally with `aptos` not having this as an export.

File: src/pages/Account/Components/TokensTable.tsx [hunk 5]
```diff
@@ -16,7 +16,9 @@ function TokenNameCell({token}: TokenCellProps) {
   return (
     <GeneralTableCell sx={{textAlign: "left"}}>
       <Link
-        to={`/token/${token?.token_data_id_hash}/${token?.property_version}`}
+        to={`/token/${token?.current_token_data?.token_data_id}/${
```
[the remark this answers is not in the document]
Comment (kent-white) [reply src/pages/Account/Components/TokensTable.tsx:19]: by broken you mean optional right?

File: src/api/hooks/useGetAccountTokens.ts [hunk 6]
```diff
@@ -1,66 +1,124 @@
...
 
-  return data?.current_token_ownerships ?? [];
+export function useGetTokenActivities(
+  tokenDataId: string,
+  limit: number,
+  offset?: number,
+) {
+  const [state] = useGlobalState();
+  return useQuery(
+    ["token_activities", {tokenDataId}, state.network_value],
```
[the remark this answers is not in the document]
Comment (kent-white) [reply src/api/hooks/useGetAccountTokens.ts:105]: good question. I was just following the pattern, but I don't think it makes sense

File: src/pages/Token/Component/ActivitiesTable.tsx (hunk 4 above)
Comment (kaw2k) [reply src/pages/Token/Component/ActivitiesTable.tsx:145]: Followed up offline, workign now

**Review by kaw2k — APPROVED** [review]

**Pull request merged**